frontend/src/ui/Footer.js

```diff
@@ -6,7 +6,7 @@ import Socials from './Socials'
 import { footerHeight, headerBackgroundImage } from './styles/constants'
 import { fullWidth } from './styles/mixins'
 
-const BaseFooter = glamorous.div(fullWidth, {
+const BaseFooter = glamorous.div([fullWidth, 'footer'], {
```

Comment:
Maybe not... I used it only for the first `expect`

Reply:
I'm not asking why did you add the class. I'm saying that this is the only place where an array is used instead of just listing the arguments. We should use the API consistently even if glamorous accepts different ways of specifying the parameters.

But maybe there was a reason to use it this way here, that was the question.

Reply:
Ok, now I understand the question. Based on what I read about Glomorous, I understood that the way to indicate two or more classes is with an array, and when it is only one class it can be listed or use an array.

Reply:
I can't find anything about that in the documentation. Can you give me a link?

Reply:
I don't remember where I had read it, or if I just tried and it worked... but I only did it for that test... in any case we can remove it if you think so.